provision.sh

```diff
@@ -11,7 +13,7 @@ export DEBIAN_FRONTEND=noninteractive
 sudo debconf-set-selections <<< "postfix postfix/main_mailer_type string 'Internet Site'"
 sudo debconf-set-selections <<< "postfix postfix/mailname string debexpo-dev"
 sudo apt-get update
-sudo apt-get install postfix python-lxml libapt-pkg-dev python-pip python-dev python-virtualenv --yes
+sudo apt-get install postfix python-lxml libapt-pkg-dev python-pip python-dev python-virtualenv python-django=1.7.1-1~bpo70+1 --yes
```

Comment:
Interesting! I never use the `=` apt-get parameter.

I usually use apt preferences instead. Admittedly parts of my Debian knowledge are stuck in 2004.